src/app/tests/SelectionsMidpoint.test.ts

```diff
@@ -0,0 +1,61 @@
+import {V} from "Vector";
+import {CreateCircuit} from "digital/public";
+
+import "./Extensions";
+
+
+describe("SelectionsMidpoint", () => {
+    test("Single Selection", () => {
+
+        // Create and place new component
+        const circuit = CreateCircuit();
+        const s1 = circuit.placeComponentAt(V(0, 0), "ANDGate");
+   
+        // Select created component
+        s1.isSelected = true;
+
+        // Check component has been selected
+        expect(s1.isSelected).toEqual(true);
+
+        // Calculate midpoint position using method
+        const sm1 = circuit.selectionsMidpoint("screen");
+        // Calculate actual midpoint position
+        const actualMidpoint = s1.pos;
```

Comment:
This is a great practice, but I think it's sufficient to just do
`expect(sm1).toEqual(s1.pos)`

Moreso, we should actually just do `expect(sm1).toEqual(V(0, 0))` since tests are really the place we *want* to be hardcoding numbers